lib/mix/lib/mix/tasks/test.ex

```diff
@@ -347,8 +347,7 @@ defmodule Mix.Tasks.Test do
 
     # Start the app and configure ExUnit with command line options
     # before requiring test_helper.exs so that the configuration is
-    # available in test_helper.exs. Then configure ExUnit again so
-    # that command line options override test_helper.exs
+    # available in test_helper.exs
```

Comment:
This is the only change that isn't purely grammatical and could probably use a bit of extra scrutiny to ensure the duplicative comments were properly divvied up